src/preprocess/vhi.py

```diff
@@ -52,6 +52,18 @@ def _preprocess(self,
         # 2. extract the timestamp for that file (from the filepath)
         timestamp = self.extract_timestamp(ds, netcdf_filepath, use_filepath=True)
 
+        # create the filepath
+        filename = self.create_filename(
+            timestamp,
+            netcdf_filepath,
+            subset_name=subset_str if subset_str is not None else None
```

Comment:
Is the end of this statement redundant? Can't this just be `subset_name=subset_str`?

Reply:
yes sorry!